packages/contracts/scripts/deploy/manager.ts

```diff
@@ -1,102 +1,119 @@
+    options: stakingShareOptions,
   },
-  CouponsForDollarsCalculator: {
-    handler: couponsForDollarsCalculatorFunc,
-    options: couponsForDollarsCalculatorOptions,
+  ubiquityChefFunc: {
+    handler: ubiquityChefFunc,
+    options: ubiquityChefOptions,
   },
-  CurveUADIncentive: {
-    handler: curveUADIncentiveFunc,
-    options: curveUADIncentiveOptions,
+  UbiquityFormulas: {
+    handler: ubiquityFormulasFunc,
+    options: ubiquityFormulasOptions,
   },
-  DebtCoupon: {
-    handler: debtCouponFunc,
-    options: debtCouponOptions,
+  SushiSwapPool: {
+    handler: sushiSwapPoolFunc,
+    options: sushiSwapPoolOptions,
   },
-  DebtCouponManager: {
-    handler: debtCouponManagerFunc,
-    options: debtCouponManagerOptions,
+
+  CreditNFT: {
+    handler: creditNFTFunc,
+    options: creditNFTOptions,
   },
-  DollarMintingCalculator: {
-    handler: dollarMintingCalculatorFunc,
-    options: dollarMintingCalculatorOptions,
+  CreditNFTManager: {
+    handler: creditNFTManagerFunc,
+    options: creditNFTManagerOptions,
   },
-  ExcessDollarDistributor: {
-    handler: excessDollarsDistributorFunc,
-    options: excessDollarsDistributorOptions,
+  CreditRedemptionCalculatorFunc: {
+    handler: creditRedemptionCalculatorFunc,
+    options: creditRedemptionCalculatorOptions,
   },
-  MasterChef: {
-    handler: masterChefFunc,
-    options: masterChefOptions,
+  CreditNFTRedemptionCalculator: {
+    handler: creditNFTRedemptionCalculatorFunc,
+    options: creditNFTRedemptionCalculatorOptions,
   },
-  UbiquityGovernance: {
-    handler: ubiquityGovernanceFunc,
-    options: ubiquityGovernanceOptions,
+  DollarMintCalculator: {
+    handler: dollarMintCalculatorFunc,
+    options: dollarMintCalculatorOptions,
   },
-  SushiSwapPool: {
-    handler: sushiSwapPoolFunc,
-    options: sushiSwapPoolOptions,
+  DollarMintExcessFunc: {
+    handler: dollarMintExcessFunc,
+    options: dollarMintExcessOptions,
   },
-  UARForDollarsCalculator: {
-    handler: uARForDollarsCalculatorFunc,
-    options: uARForDollarsCalculatorOptions,
+  TWAPOracleDollar3pool: {
+    handler: twapOracleDollar3poolFunc,
+    options: twapOracleDollar3poolOptions,
   },
-  UbiquityAlgorithmicDollar: {
-    handler: ubiquityAlgorithmicDollarFunc,
-    options: ubiquityAlgorithmDollarOptions,
+  UbiquityCreditTokenFunc: {
+    handler: ubiquityCreditTokenFunc,
+    options: ubiquityCreditTokenOptions,
   },
-  UbiquityAutoRedeem: {
-    handler: ubiquityAutoRedeemFunc,
-    options: ubiquityAutoRedeemOptions,
+  UbiquityDollarManager: {
+    handler: ubiquityDollarManagerFunc,
+    options: ubiquityDollarManagerOptions,
   },
-  UbiquityFormulas: {
-    handler: ubiquityFormulaFunc,
-    options: ubiquityFormulaOptions,
+  UbiquityDollarToken: {
+    handler: ubiquityDollarTokenFunc,
+    options: ubiquityDollarTokenOptions,
   },
-  YieldProxy: {
-    handler: yieldProxyFunc,
-    options: yieldProxyOptions,
+  UbiquityGovernanceTokenFunc: {
```

Comment:
The naming convention here is incorrect. This is likely supposed to be `UbiquityGovernanceToken`. Same with the others.

Reply:
Update: just reviewed https://github.com/ubiquity/ubiquity-dollar/issues/292#issuecomment-1288255782 and this should be `UbiquityGovernanceToken`

Reply:
good cacth fixed !